<launch>
  <node name="cons_robot_1" pkg="ros_ex" type="consensus" output="screen">
    <param name="robot_name" value="turtle1"/>
  </node>
  
  <node name="cons_robot_2" pkg="ros_ex" type="consensus" output="screen" >
    <param name="robot_name" value="turtle2"/>
  </node>
  
  <node name="cons_robot_3" pkg="ros_ex" type="consensus" output="screen" >
    <param name="robot_name" value="turtle3"/>
  </node> 
  
  
 </launch>